README.md

```diff
@@ -501,9 +501,16 @@ Example:
 
 Yes.
 
-### Does this library work with Angular 5.x?
+### Does this library work with Angular Versions < 8.0.0?
 
-The latest version that supports Angular 5.x is 1.1.1. You can still install it via npm or yarn, e.g. `npm install ngx-drag-to-select@1.1.1`.
+Please have a look here to see which version supports your angular application. 
+
+You can still install it via npm or yarn, e.g. `npm install ngx-drag-to-select@x.x.x`.
+
+| Angular Version      | ngx-drag-to-select Version |
```

Comment:
Great idea! Thanks for this addition.

Reply:
Let's change the headers to only `Angular` and `ngx-drag-to-select`. The "version" can be inferred by the user very quickly by looking at the rows.